CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### App Center
 
+* **[Fix]** Fix converting string properties to chars after converting to UTF-8.
```

Comment:
Please re-word. As a user I don't understand what is it and how it affects my application.
It should include that:
- It's iOS only issue;
- It affects the apps only if it uses non-ASCII chars
- It should describe use cases, but not what the changed line does